app/steps/ui/will/codicilsnumber/index.js

```diff
@@ -14,14 +15,30 @@ class CodicilsNumber extends ValidationStep {
         return ctx;
     }
 
-    handlePost(ctx, errors) {
+    handlePost(ctx, errors, formdata, session, hostname, featureToggles) {
```

Comment:
Can we have a unit test for at least the new variable that was added to this method